packages/maskbook/src/plugins/ITO/UI/ITO.tsx

```diff
@@ -763,10 +782,7 @@ function ITO_LoadingFailUI({
 
 export function ITO_ConnectMetaMask() {
     return (
-        <ITO_LoadingFailUI
-            retryPoolPayload={async () => await Services.Ethereum.connectMetaMask()}
-            isConnectMetaMask={true}
-        />
+        <ITO_LoadingFailUI retryPoolPayload={async () => await Services.Ethereum.connectMetaMask()} isConnectMetaMask />
```

Comment:
extract `retryPoolPayload` property as variable
```suggestion
        const onRetry = () => Services.Ethereum.connectMetaMask()
        return <ITO_LoadingFailUI retryPoolPayload={onRetry} isConnectMetaMask />
```

Reply:
Maybe another PR.

Reply:
```suggestion
        <ITO_LoadingFailUI retryPoolPayload={() => Services.Ethereum.connectMetaMask()} isConnectMetaMask />
```